hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -208,14 +215,13 @@ module top_${top["name"]} #(
 % endfor
 
   // Clock assignments
-% for clock in top['clocks']:
-  % if clock['name'] != "usb" :
-  assign ${clock['name']}_clk = clk_i;
-  % endif
+  // These assignments are temporary until the creation of the clock controller
+% for clock_group in clks_attr['groups']:
```

Comment:
@eunchan 
i decided to go a little further here and just do what i would be doing inside the clock controller.
This also means the `clk_fixed_i` port is added.

I know we've talked about different names, and once I get confirmation from the partners, I'll push a separate PR to get that all lined up.

Reply:
SG